### PR TITLE
New feature additions to User Guide.

### PR DESCRIPTION
**Pull request by aditi2313**
1.` patron overdue` command additions.
2. Merged other newly added patron and book commands.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -645,10 +645,10 @@ testers are expected to do more *exploratory* testing.
 
    1. Test case: `book delete 1`<br>
       Expected: First book is deleted from the list. Details of the deleted book shown in the status message. Timestamp in the status bar is updated.
-   
+
    1. Test case: `book delete 0`<br>
       Expected: No book is deleted. Error details shown in the status message. Status bar remains the same.
-   
+
```
Comment (Halpfrog) [inline docs/DeveloperGuide.md:651]: LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: page
 title: User Guide
 ---
 
-LibTask is a **desktop application for librarians to manage book loans and requests by patrons, optimized for use via a Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your book tracking tasks done faster than traditional GUI apps. 
+LibTask is a **desktop application for librarians to manage book loans and requests by patrons, optimized for use via a Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). If you can type fast, LibTask can get your book tracking tasks done faster than traditional GUI apps.
```
Comment (Halpfrog) [inline docs/UserGuide.md:6]: Ensures that all mentions of AB3 is no longer in our UG

File: docs/team/yl-ang.md [hunk 3]
```diff
@@ -32,6 +32,5 @@ Given below are my contributions to the project.
 
 * **Community**:
   * PR reviewed (with non-trivial review comments): [\#101](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/101), [\#100](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/100), [\#96](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/96), [\#95](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/95), [\#83](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/83), [\#79](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/79), [\#78](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/78), [\#73](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/73), [\#69](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/69), [\#66](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/65), [\#62](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/65), [\#60](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/60)
-  *
 
-* **Tools**:
\ No newline at end of file
+* **Tools**:
```
Comment (Halpfrog) [inline docs/team/yl-ang.md:36]: Gets rid of extra whitespace

File: docs/team/yuqitanyq.md [hunk 4]
```diff
@@ -63,11 +60,11 @@ Given below are my contributions to the project.
     * user story for EditBookCommand
     * user story for ListBookCommand
     * logic illustration of EditBookCommand, ListBookCommand and RelatedBookCommand.
-  
+
 
 * **Community**:
   * Raised issues on the forum if necessary and gave detailed descriptions of problems.
-  
+
```
Comment (Halpfrog) [inline docs/team/yuqitanyq.md:68]: Gets rid of extra whitespace

**Review by Halpfrog — APPROVED** [review]

**Review by yl-ang — APPROVED** [review]
LGTM!

**Pull request merged**